frontend/src/composables/useTerminalSplitpanes.js

```diff
@@ -183,72 +153,57 @@ export const useTerminalSplitpanes = ({ name, namespace, target }) => {
     leavePageIfTreeEmpty()
   }
 
-  function getLocalStorageObject (key) {
-    return useLocalStorage(key).value
-  }
-
-  function setLocalStorageObject (key, value) {
-    useLocalStorage(key).value = value
-  }
-
   function updateSplitpaneTree () {
-    state.splitpaneTree = state.tree.toJSON(state.tree.root)
-    saveSplitpaneTree()
+    splitpaneTree.value = symbolTree.toJSON(symbolTree.root)
+
+    const onySlotItemsInTree = every(symbolTree.ids(), id => includes(slotItemUUIds.value, id))
+    localStorageStore.terminalSplitpaneTree = onySlotItemsInTree || symbolTree.isEmpty()
+      ? null // clear value
+      : splitpaneTree.value
   }
 
   function targetIdOrDefault (targetId) {
     if (!targetId) {
-      targetId = focusedElementId.value // TODO check if it still works
+      targetId = appStore.focusedElementId // TODO check if it still works
```

Comment:
```suggestion
      targetId = appStore.focusedElementId
```
we can remove this comment

Reply:
✔️